bff/samples/Hosts.Tests/TestInfra/AppHostFixture.cs

```diff
@@ -1,6 +1,8 @@
 using Aspire.Hosting;
 using Microsoft.Extensions.Logging;
+#if !DEBUG_NCRUNCH
```

Comment:
The default editorconfig prefers using usings over fully qualified names. But this 'Projects' node is only present in aspire, not when building under ncrunch. So applying condional compilation here.